generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -212,3 +213,26 @@ class LBWIFRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithNoMemPort ++       // remove AXI4 backing memory
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
   new chipyard.config.AbstractConfig)
+
+class MulticlockAXIOverSerialConfig extends Config(
+  new chipyard.config.WithFbusToSbusCrossingType(RationalCrossing(SlowToFast)) ++
+
+  new chipyard.config.WithSystemBusFrequencyAsDefault ++
+  new chipyard.config.WithSystemBusFrequency(4000) ++
+  new chipyard.config.WithPeripheryBusFrequency(4000) ++
+  new chipyard.config.WithMemoryBusFrequency(4000) ++
+
+  new chipyard.config.WithFrontBusFrequency(4000 / 2) ++
+
+  new chipyard.config.WithFbusToSbusCrossingType(RationalCrossing(SlowToFast)) ++
+  new testchipip.WithAsynchronousSerialSlaveCrossing ++
+
+  new chipyard.harness.WithSimAXIMemOverSerialTL ++ // add SimDRAM DRAM model for axi4 backing memory over the SerDes link, if axi4 mem is enabled
+  new chipyard.iobinders.WithSerialTLAndPassthroughClockPunchthrough ++ // add new clock for axi domain over serdes and passthrough ios
+  new chipyard.config.WithSerialTLBackingMemory ++ // remove axi4 mem port in favor of SerialTL memory
+  new testchipip.WithBlockDeviceLocations(
+    freechips.rocketchip.subsystem.PBUS,
```

Comment:
Does this work? The block device can't master memory if both edges connect to the pbus

Reply:
This doesn't work so far. I hadn't really looked into why the block device is failing past this though. As you mentioned, I just decided to remove the block device in an unpushed commit (more closely matches a chip config where there would instead be a "SPIBridge" of some sorts).